processing/src/main/java/org/apache/druid/query/rowsandcols/semantic/DefaultFramedOnHeapAggregatable.java

```diff
@@ -719,7 +719,7 @@ private AppendableRowsAndColumns aggregateWindowApertureWellBehaved(
     // End Phase 3, anything left in the window needs to be collected and put into our results
     for (; nextIndex < windowSize; ++nextIndex) {
       for (int i = 0; i < aggFactories.length; ++i) {
-        results[i][resultStorageIndex] = aggregators[i][nextIndex].get();
```

Comment:
is `aggregators[i][nextIndex].close();` missing here?

Reply:
yes, added it